Modules/Filtering/FFT/src/itkFFTWFFTImageFilterFactory.cxx

```diff
@@ -36,55 +36,47 @@
 
 namespace itk
 {
-// Undocumented API used to register during static initialization.
-// DO NOT CALL DIRECTLY.
-void ITKFFT_EXPORT
-     FFTWComplexToComplex1DFFTFactoryRegister__Private()
+FFTWFFTImageFilterFactory::FFTWFFTImageFilterFactory()
 {
   ObjectFactoryBase::RegisterInternalFactoryOnce<FFTImageFilterFactory<FFTWComplexToComplex1DFFTImageFilter>>();
-}
-
-void ITKFFT_EXPORT
-     FFTWComplexToComplexFFTFactoryRegister__Private()
-{
   ObjectFactoryBase::RegisterInternalFactoryOnce<FFTImageFilterFactory<FFTWComplexToComplexFFTImageFilter>>();
-}
-
-void ITKFFT_EXPORT
-     FFTWForward1DFFTFactoryRegister__Private()
-{
   ObjectFactoryBase::RegisterInternalFactoryOnce<FFTImageFilterFactory<FFTWForward1DFFTImageFilter>>();
-}
-
-void ITKFFT_EXPORT
-     FFTWForwardFFTFactoryRegister__Private()
-{
   ObjectFactoryBase::RegisterInternalFactoryOnce<FFTImageFilterFactory<FFTWForwardFFTImageFilter>>();
-}
-
-void ITKFFT_EXPORT
-     FFTWHalfHermitianToRealInverseFFTFactoryRegister__Private()
-{
   ObjectFactoryBase::RegisterInternalFactoryOnce<FFTImageFilterFactory<FFTWHalfHermitianToRealInverseFFTImageFilter>>();
+  ObjectFactoryBase::RegisterInternalFactoryOnce<FFTImageFilterFactory<FFTWInverse1DFFTImageFilter>>();
+  ObjectFactoryBase::RegisterInternalFactoryOnce<FFTImageFilterFactory<FFTWInverseFFTImageFilter>>();
+  ObjectFactoryBase::RegisterInternalFactoryOnce<FFTImageFilterFactory<FFTWRealToHalfHermitianForwardFFTImageFilter>>();
```

Comment:
Per above discussion, this is where `FFTImageFilterFactory` is actually the factory being used to accomplish overrides.

Reply:
@thewtex Follow-up question to earlier discussion, is the use of magic statics in initialization causing Slicer crashes? `ObjectFactoryBase::RegisterInternalFactoryOnce` has been updated to rely on magic statics, so if this is the case then we will need to use a different mechanism inside any `InitFactory` objects that we expect to be registered in Python modules.

Reply:
> magic statics in initialization causing Slicer crashes?

Yes. :-(

> we will need to use a different mechanism inside any InitFactory objects that we expect to be registered in Python modules.

Right. Currently the patch is calling the `RegisterOneFactory` method in Python instead of using magic statics.